src/components/services/pathutil.spec.ts

```diff
@@ -3,12 +3,28 @@ import "angular-mocks"
 
 import {PathUtil} from "./pathutil";
 
-describe("Path-Util", () => {
+describe("PathUtil", () => {
 
    it("should throw error if passed undefined or null in toPropertyAccessString", () => {
       expect(() => PathUtil.toPropertyAccessString(undefined)).toThrowError();
       expect(() => PathUtil.toPropertyAccessString(null)).toThrowError();
    });
+
+    it("should return an empty array when converting a path to fragments", () => {
+        expect(PathUtil.toPropertyFragments(undefined)).toEqual([]);
+    });
+
+    it("should return the last fragment", () => {
+        expect(PathUtil.lastFragment("foo/bar")).toBe("bar")
+    });
+
+    it("should beautify the last fragment", () => {
+        expect(PathUtil.beautifiedLastFragment("foo/bar")).toBe("Bar")
+    });
+
+    it("should filter indices from a path", () => {
+        expect(PathUtil.filterIndexes("foo/items/1/bar")).toBe("foo/items/bar")
+    });
    
```

Comment:
a test case for a beautify of a property like firstName to be "First Name" would be good